src/MensaWidget/MWFrame.class.st

```diff
@@ -77,66 +62,55 @@ MWFrame >> buildFrameContent [
 	self fillContentMorph: contentMorph.
 	contentMorph scrollBarThickness: self class scrollBarThickness;
 		color: self class contentMorphColor;
-		position: self class contentPosition;
 		extent: self class contentExtent.
-	self addPaneMorph: contentMorph
+	^ contentMorph
 ]
 
 {
 	#category : #creation,
-	#'squeak_changestamp' : 'Louis Christ 6/7/2024 11:05'
+	#'squeak_changestamp' : 'Oscar 6/14/2024 12:49'
 }
 MWFrame >> fillContentMorph: aContentMorph [
 	
 	| contentMorph |
 	
 	contentMorph := AlignmentMorph newColumn
 		cellInset: self class cellSpacing;
-		hResizing: #shrinkWrap;
-		vResizing: #shrinkWrap.
+		extent: self class contentExtent;
+		hResizing: #spaceFill;
+		vResizing: #spaceFill.
```

Comment:
I think it would be nice to have an extra method that returns the alignment morph with the desired properties